lib/pages/community.dart

```diff
@@ -354,7 +360,9 @@ class _CommunityOverview extends StatelessWidget {
                           padding: EdgeInsets.only(right: 3),
                           child: Icon(Icons.record_voice_over, size: 20),
                         ),
-                        const Text('xx'), // TODO: display online users
+                        Text(onlineUsers == null
+                            ? 'xx'
```

Comment:
there's no need for ternary since it's never going to be null

https://github.com/LemmyNet/lemmy/blob/cf911c023df320d43d631191cb1de6f017158ea1/crates/structs/src/community.rs#L20

Reply:
It can since it is being fetched and passed to this widget as nullable (hence no null assert check in the constructor)

It's null if it has not been fetched yet

Reply:
oh, thats fine then

Reply:
oh, ok then